advocacy_docs/supported-open-source/patroni/installing_patroni.mdx

```diff
@@ -7,28 +7,49 @@ tags:
   - Patroni
 ---
 
-Patroni packages are provided through the PGDG `apt` and `yum` repositories.
+EDB provides Patroni to customers via the `edb-patroni` package. This package provides Patroni itself and all its dependencies, so there is no need to install additional packages on the Postgres nodes.
 
-See [Platform Compatibility](https://www.enterprisedb.com/resources/platform-compatibility#epas) for the supported OS list (only `Linux x86-64 (amd64)` currently).
+Extra packages which no longer have to be installed separately are the following:
 
-### Debian/Ubuntu
+- python3-cdiff
+- python3-psutil
+- python3-psycopg2
+- python3-ydiff
+- python3-click
+- python3-six
+- python3-dateutil
+- python3-prettytable
+- python3-pyyaml
+- python3-urllib3
+- python3-etcd
+- python3-dns
+- python3-certifi
+
+Packages are available for all subscribed customers with a valid EDB account under any entitlement (Community360, Standard, and Enterprise).
+
+!!! Note
+    The `edb-patroni` does not provide packages for the etcd server needed for the DCS cluster
 
-To install Patroni, configure the [PostgreSQL](https://www.postgresql.org/download/linux/debian/) `apt` repository. Then run:
+Once you have the EDB repository configured on all the nodesof the cluster, run the following commands depending on the Linux distribution you are using
+
+### Debian/Ubuntu
 
 ```bash
-sudo apt-get install -y patroni
+sudo apt-get install -y edb-patroni
 ```
 
+!!! Note
+    On Debian and Ubuntu installations, if you've previously installed the Patroni package named `patroni` using the EDB repositories, `apt upgrade` will not replace this package with the `edb-patroni` package. Executing `apt install edb-patroni` will install `edb-patroni` as a replacement of `patroni`.
+
 See [Quick start on Debian 11](debian11_quick_start/#4-patroni) for a more detailed configuration example.
 
 ### RHEL/CentOS
 
-You can install Patroni from the [PostgreSQL](https://yum.postgresql.org/) `yum` repository. It requires Extra Packages for Enterprise Linux ([EPEL](https://fedoraproject.org/wiki/EPEL)).
-
-After you configured the repositories, run the following command to install Patroni and its dependencies for etcd:
-
 ```bash
-sudo dnf install -y patroni patroni-etcd
+sudo dnf install -y edb-patroni
 ```
 
 See [Quick start on RHEL8](rhel8_quick_start/#4-patroni) for a more detailed configuration example.
+
+
+We also support community packages provided via the PGDG repositories. Follow the [PGDG deb download instructions](https://www.postgresql.org/download/linux/debian/) to set up the `apt` repository, or the [PGDG rpm download instructions](https://yum.postgresql.org/) for the `yum` repository. Keep in mind that for PGDG rpm repositories you will need to configure Extra Packages for Enterprise Linux ([EPEL](https://fedoraproject.org/wiki/EPEL)) and enable the `pgdg-extras` repository.
```

Comment:
I wonder if we should add a different heading here for the notes?
I mean, it looks like it's part of the RedHat section, while the note is important both for Debian and RedHat based.

Something else that came to my mind: we are missing instructions for SLES in the docs.

Reply:
Maybe I should move it to the head of this section, before going into details on Debian and RHEL installations. Or create a PGDG section